docs/guide/basics/examples/size-icon-font-example/files.ts

```diff
@@ -0,0 +1,21 @@
+import App from './App.js?raw'
+import styles from '../styles.css?raw'
+import IconCss from './icon.css?raw'
+
+const files = {
+  'icon.css': {
+    code: IconCss,
+    readOnly: false,
+    active: true,
+  },
+  'App.js': {
+    code: App,
+  },
+  'styles.css': {
+    code:styles,
```

Comment:
Seems to be a common theme here that there is no space between `code:` and `styles` ^^

Reply:
Oh tnx, didn't noticed will clean that up!